install.sh

```diff
@@ -96,7 +96,7 @@ function verifyCLI() {
 
```

Comment:
@dlm I think there is a bug on line 90. Since the file that was downloaded was marked as executable with `chmod +x` on line 78, line 90 will always succeed.

Reply:
I updated the curl commands to fail with 400+ status codes and went with `--help` to verify the cli works.

Reply:
sounds good.